regions/__init__.py

```diff
@@ -12,4 +12,6 @@
 
 # For egg_info test builds to pass, put package imports here.
 if not _ASTROPY_SETUP_:
-    pass
+    import io
+    import shapes
+    import core
```

Comment:
Needs changing to e.g. ``from . import io`` since these kinds of absolute imports don't work properly in Python 3?